js/models/account.js

```diff
@@ -74,6 +75,17 @@ define(function(require) {
 			}
 			return undefined;
 		},
+		getSpecialFolder: function() {
+			if (!this.folders) {
+				return undefined;
+			}
+			return _.find(this.folders, function(folder) {
+				// TODO: handle special folders in subfolder properly
```

Comment:
Maybe use `@TODO` so documentation tools can pick it up? It is very common though.